config/locales/en/browse.yml

```diff
@@ -2,16 +2,54 @@
 en:
   browse:
     popular_links:
+      abroad:
+        variant_a:
+          - title: Get access to your eVisa
+            url: /get-access-evisa
+          - title: Track your passport application
+            url: /track-passport-application
+          - title: Online immigration status (eVisa)
+            url: /guidance/online-immigration-status-evisa
+        variant_b:
+          - title: Apply online for a UK passport
+            url: /apply-renew-passport
+          - title: Foreign travel advice
+            url: /foreign-travel-advice
+          - title: Get a passport urgently
+            url: /get-a-passport-urgently
       benefits:
         control:
           - title: Check benefits and financial support you can get
             url: /check-benefits-financial-support
         variant_a:
-          - title:
-            url:
+          - title: Universal Credit
+            url: /universal-credit
+          - title: Sign in to your Universal Credit account
+            url: /sign-in-universal-credit
+          - title: Child Benefit
+            url: /child-benefit/how-to-claim
```

Comment:
In the spreadsheet it's /child-benefit

Reply:
Oh good spot Mat - thank you so much 🕵️  I will amend

Reply:
Done